test-e2e/test/helpers.js

```diff
@@ -109,11 +113,23 @@ async function genFixture(...fixture: Fixture) {
   return {rootPath, binPath, projectPath, esy, esyPrefixPath};
 }
 
+function skipSuiteOnWindows(blockingIssues) {
+   if (process.platform === 'win32') {
+      fdescribe("", () => {
+         fit('does not work on Windows', () => {
```

Comment:
heh, I was surprised jest don't have some utility to conditinoally skip test cases. This one is a cool trick!

Reply:
The weird part is that this is from the jest source and they use it in their own tests

Reply:
Yep, shamelessly stolen from the `jest source`:
https://github.com/facebook/jest/blob/84f0ea307e65c1d8a530781aeab06da4cd5a97a1/scripts/ConditionalTest.js
😄